### PR TITLE
Adding AirVPN.pkg v2.8 Cask

### PR DESCRIPTION
**Pull request by moogar0880**
Adds support for installing the [AirVPN OS X](https://airvpn.org/macosx/) vpn client

### CODE REVIEW

**Comment by vitorgalvao** [comment]
It’s missing `uninstall`. There are [scripts](https://github.com/caskroom/homebrew-cask/tree/master/developer/bin) and [documentation](https://github.com/caskroom/homebrew-cask/blob/master/doc/CASK_LANGUAGE_REFERENCE.md#uninstall-stanza-details) geared for that. Do you need any help?

File: Casks/airvpn.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+cask :v1 => 'airvpn' do
+  version '2.8'
+  sha256 '311babbdecef814a8c6168ee625990338489dde973bef83b3fe157db493a1cef'
+
+  url "https://airvpn.org/repository/#{version}/airvpn_osx_x64_installer.pkg"
+
```
Comment (vitorgalvao) [inline Casks/airvpn.rb]: This empty line shouldn't be here.

**Comment by moogar0880** [comment]
Ah, sorry, missed that completely. I'll take a crack at writing the `uninstall` logic

**Comment by moogar0880** [comment]
Not entirely sure what happened with this PR...I'm going to close it and open a new one

**Pull request closed without merging**